src/components/sidebar/right-sidebar/Filters.tsx

```diff
@@ -100,12 +111,15 @@ export default function Filters({ page }: FilterProps) {
           onChange={(e: any, data: any) => {
             addFilter(data.value, filter_type)
           }}
+          value={filters[filter_type] as string[]}

```

Comment:
Curious: Is this the thing that enables u to clear the visible filters as well (The fix to the million api calls issue?)